### ISSUE TITLE
Disabling permission to access specific nodes doesn't work

### ISSUE THREAD

**Issue by r15ch13** [Valandur/Web-API#21]
Sponge: `spongevanilla-1.11.2-6.1.0-BETA-16.jar`
WebAPI: `webapi-4.2.2-S6.1.jar`

I can't get the default from [PERMISSIONS.md](https://github.com/Valandur/Web-API/blob/master/docs/PERMISSIONS.md) to work. 
```
# These are the default permissions that a client without a key receives
default {
    # The permissions define which endpoints a user with this key can access, and what data is
    # returned for each endpoint
    permissions = {
        info = "*"
        player = {
            get = {
                "*" = true
                uuid = false
            }
        }
    }

    # The rate limit specifies how many operations per second a client with this key can execute
    rateLimit = 10
}
```
Results in 

```
{
  "code": 403,
  "message": "Not authorized"
}
```
Separating `get` and `post` permissions results in `403` everytime.
When I remove `get` it lets me access the player data, but it still returns the `uuid`.

Permissions I would like to use:
```
{
    key=OVERVIEWER
    permissions {
        get {
            info {
                "*"=true
            }
            player {
                "*"=true
                connection {
                    address=false
                }
                inventory=false
                achievements=false
                statistics {
                    "*"=false
                    deaths=true
                }
            }
        }
        post { 
            "*" = false
        }
    }
    rateLimit=1000
}
```

**Comment by Valandur** [comment]
I'm afraid I renamed the permissions in one of the newest releases, but this change might not have been documented properly yet.
It should be correct in the endpoint documentation provided at http://localhost:8080/docs, or [here](https://valandur.github.io/Web-API/redoc.html), but probably wrong in some of the tutorials.
The permission node for getting a single player would be `one`, and for getting the list of players `list`. Updating a player would be done with `change`, so your permissions should look something like this:
```
{
    key=OVERVIEWER
    permissions {
        list {
        one {
            info {
                "*"=true
            }
            player {
                "*"=true
                connection {
                    address=false
                }
                inventory=false
                achievements=false
                statistics {
                    "*"=false
                    deaths=true
                }
            }
        }
        change=false
    }
    rateLimit=1000
}
```

**Comment by r15ch13** [comment]
Ah okay, there is no way I could have figured that out then :grin:
But something is still wrong I guess:
The `address` is still there (and just for testing `armour` doesn't work also)
```
// 20170724162011
// http://local.streuer.io:8085/api/player/062b8c09-4ef8-4798-91a0-29f92b1c8d41?key=OVERVIEWER

{
  "ok": true,
  "player": {
    "uuid": "xxxx",
    "name": "r15ch13",
    "type": "minecraft:player",
    "link": "/api/player/xxxx",
    "location": {
      "world": {
        "uuid": "xxxx",
        "name": "vanilla2017",
        "link": "/api/world/xxxx"
      },
      "position": {
        "x": -312.4918009877637,
        "y": 79.5,
        "z": -0.5332617601833258
      }
    },
    "class": "valandur.webapi.api.cache.player.CachedPlayer",
    "rotation": {
      "x": 1.935563087463379,
      "y": 237.86318969726562,
      "z": 0.0
    },
    "velocity": {
      "x": 0.0,
      "y": -0.0784000015258789,
      "z": 0.0
    },
    "scale": {
      "x": 1.0,
      "y": 1.0,
      "z": 1.0
    },
    "connection": {
      "latency": 0,
      "address": "/xxxxxxxx:47753"
    },
    "armour": {
      "helmet": null,
      "chestplate": {
        "id": "minecraft:iron_chestplate",
        "name": "Iron Chestplate",
        "quantity": 1,
        "data": {
          "durability": {
            "unbreakable": false,
            "durability": 55
          }
        }
      },
      "leggings": {
        "id": "minecraft:iron_leggings",
        "name": "Iron Leggings",
        "quantity": 1,
        "data": {
          "durability": {
            "unbreakable": false,
            "durability": 41
          }
        }
      },
      "boots": {
        "id": "minecraft:iron_boots",
        "name": "Iron Boots",
        "quantity": 1,
        "data": {
          "durability": {
            "unbreakable": false,
            "durability": 15
          }
        }
      }
    },
    "joined": {
      "first": 1493554626,
      "last": 1500906004
    },
    "health": {
      "current": 20.0,
      "max": 20.0
    },
    "experience": {
      "level": 65,
      "experience": 9,
      "totalExperience": 11030
    },
    "food": {
      "foodLevel": 18,
      "exhaustion": 1.428391933441162,
      "saturation": 0.0
    },
    "gameMode": {
      "id": "minecraft:survival",
      "name": "survival"
    },
    "statistics": {
      "deaths": 8
    }
  }
}
```

```{
        key=OVERVIEWER
        permissions {
            info="*"
            player {
                one {
                    "*"=true
                    connection {
                        "*"=false
                        latency=true
                    }
                    inventory=false
                    achievements=false
                    armour=false
                    statistics {
                        "*"=false
                        deaths=true
                    }
                }
                list {
                    "*"=true
                    connection {
                        "*"=false
                        latency=true
                    }
                    inventory=true
                    achievements=false
                    armour=false
                    statistics {
                        "*"=false
                        deaths=true
                    }
                }
            }
            change=false
        }
        rateLimit=1000
    }```

**Comment by Valandur** [comment]
This is fixed in the newest release [v0.4.3](https://github.com/Valandur/Web-API/releases/tag/v4.3.0)
